### PR TITLE
[PHP] Tests if const is defined

### PR DESCRIPTION
**Pull request by toutma**

### CODE REVIEW

File: src/Service/WordpressHelper.php [hunk 1]
```diff
@@ -168,7 +168,10 @@ public function registerStylesheets()
 
     public function catchAllAction()
     {
-        define('WP_USE_THEMES', true);
+        if(! defined('WP_USE_THEMES')) {
```
Comment (zalexki) [inline src/Service/WordpressHelper.php]: This is not JS, please do PSR-2 at least :)

Comment (zalexki) [inline src/Service/WordpressHelper.php]: This is not JS, please do PSR-2 at least :)

File: src/Service/WordpressHelper.php [hunk 2]
```diff
@@ -168,7 +168,10 @@ public function registerStylesheets()
...
+            define('WP_USE_THEMES', true);
+        }
+        
         $response = $this->getResponse();
```
Comment (zalexki) [inline src/Service/WordpressHelper.php]: useless variable assignation, you can directly `return $this->getResponse();`

Comment (toutma) [reply src/Service/WordpressHelper.php]: 👍🏻

File: src/Service/WordpressHelper.php (hunk 1 above)
Comment (toutma) [reply src/Service/WordpressHelper.php]: 🤭

**Pull request merged**